src/Test/TestState.php

```diff
@@ -78,6 +78,7 @@ public static function disableDefaultProxyAutoRefresh(): void
 
     /**
      * @deprecated Use TestState::enableDefaultProxyAutoRefresh()
+     * @noinspection PhpDeprecationInspection
```

Comment:
I don't really understand why we should hide that we're using a deprecated method?

Reply:
I have reverted it